src/SpikeCore/SpikeCore.Web/Startup.cs

```diff
@@ -130,7 +137,8 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
                 .SingleInstance();
 
             containerBuilder
-                .RegisterAssemblyTypes(AppDomain.CurrentDomain.GetAssemblies().Single(assembly => assembly.GetName().Name == "SpikeCore"))
+                .RegisterAssemblyTypes(AppDomain.CurrentDomain.GetAssemblies()
+                    .Single(assembly => assembly.GetName().Name == "SpikeCore"))
```

Comment:
Since you dared to reformat this line, i can also criticize it :trollface:, why not just
```suggestion
                .RegisterAssemblyTypes(typeof(IModule).Assembly)
```

Reply:
Ah, good idea. Please do feel free to bring up other points about things we can improve: it's not criticism, it's constructive feedback. Smippy and I wrote this mainly in our spare time, and may have uh... cut a few corners. I also don't write much C# lately, so I'm pretty sure you'll find all kinds of dumb things I've done, heh.